cerberus-audit-logger-athena/src/test/java/com/nike/cerberus/audit/logger/AthenaClientFactoryTest.java

```diff
@@ -0,0 +1,16 @@
+package com.nike.cerberus.audit.logger;
+
+import com.amazonaws.services.athena.AmazonAthena;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class AthenaClientFactoryTest {
+
+  @Test
+  public void testGetClientAlwaysReturnsSameAthenaInstance() {
+    AthenaClientFactory athenaClientFactory = new AthenaClientFactory();
+    AmazonAthena clientInstance1 = athenaClientFactory.getClient("region-2");
```

Comment:
What happens if this region is null or an empty string? Can you add tests for if `getClient` is called with bad parameters?

Reply:
![Test](https://user-images.githubusercontent.com/80131455/113942192-4c4e3280-97b5-11eb-9c4f-43c604d71d0a.png)
the test is failing as shown in the image , i dont know what the alphanumeric  it's not matching with expected even i passed empty string at both places. Please take a look at this pic

Reply:
Code is not throwing an exception for bad parameters.